config/deploy-settings.xml

```diff
@@ -1,7 +1,7 @@
 <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
-                              http://maven.apache.org/xsd/settings-1.0.0.xsd">
+                              https://maven.apache.org/xsd/settings-1.0.0.xsd">
```

Comment:
Can we update xsi:schemaLocation also?

Reply:
What should be the value of xsi:schemaLocation

Reply:
Https

Reply:
@romani xsi:schemaLocation is to be changed for all files or for file only

Reply:
For all files

Reply:
@romani 
Error "cvc-elt.1.a: Cannot find the declaration of element 'project'." 
is shown on vs code while changing value of xsi:schemaLocation

Reply:
Ok, sad .

Please fix commits, by squashing all in one to let me merge this Place